image/manifest_test.go

```diff
@@ -0,0 +1,67 @@
+package image
+
+import (
+	"testing"
+
+	"github.com/containers/image/manifest"
+	"github.com/containers/image/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestManifestLayerInfosToBlobInfos(t *testing.T) {
+	blobs := manifestLayerInfosToBlobInfos([]manifest.LayerInfo{})
+	assert.Equal(t, []types.BlobInfo{}, blobs)
+
+	blobs = manifestLayerInfosToBlobInfos([]manifest.LayerInfo{
+		{
+			BlobInfo: types.BlobInfo{
+				MediaType: "application/vnd.docker.image.rootfs.diff.tar.gzip",
```

Comment:
Not a blocker: this can be `manifest.DockerV2Schema2LayerMediaType`.

Reply:
Yes, that would minimize the repetition.

On balance, I think I prefer the tests of interoperability with external protocols to hard-code values, at least some of the time, so that they detect unwanted changes to definitions of these constants — but it’s a weak preference, the repetition is not great either.

In this case it might even make more sense to use `"a"`,`"b"`,`"c"` or something similar to ensure that the values are passed through unchanged in the right order, but that would be even further from real-world usage.